build/stf-run-ci/tasks/setup_base.yml

```diff
@@ -59,7 +68,7 @@
           name: openshift-cert-manager-operator
           namespace: openshift-cert-manager-operator
         spec:
-          channel: tech-preview
+          channel: "{{ cert_manager_channel }}"
```

Comment:
Shouldn't we have a value for OCP >= 4.12? Not sure what is the behavior if this string is empty

Reply:
The default value is set in defaults/main.yml L61 (see above)